treeherder/etl/jobs.py

```diff
@@ -335,12 +335,17 @@ def _schedule_log_parsing(job, job_logs, result):
     # RabbitMQ priority feature (since the idea behind separate queues was
     # only to ensure failures are dealt with first if there is a backlog).
     if result != 'success':
-        queue = 'log_parser_fail'
-        priority = 'failures'
+        if job_lab.name == "errorsummary_json":
+            queue = "log_parser_fail_json"
+            priority = "failures"
+        else:
+            queue = "log_parser_fail_raw"
+            priority = "failures"
     else:
         queue = 'log_parser'
         priority = "normal"
 
+    # XXX I think I'll change this to turn parse_logs into non-Task?
```

Comment:
Right, then the async tasks will get assigned to queues one step lower as you mentioned in `tasks.py`